openhands/runtime/utils/bash.py

```diff
@@ -189,13 +189,15 @@ def initialize(self):
         if self.username in ['root', 'openhands']:
             # This starts a non-login (new) shell for the given user
             _shell_command = f'su {self.username} -'
-        # otherwise, we are running as the CURRENT USER (e.g., when running LocalRuntime)
-        if self.max_memory_mb is not None:
-            window_command = (
-                f'prlimit --as={self.max_memory_mb * 1024 * 1024} {_shell_command}'
-            )
-        else:
-            window_command = _shell_command
+
+        # FIXME: we will introduce memory limit using sysbox-runc in coming PR
```

Comment:
let's just delete the code--it's not hard to go back into git history to recover it